framework/db/Command.php

```diff
@@ -537,6 +541,7 @@ public function createTable($table, $columns, $options = null)
     public function renameTable($table, $newName)
     {
         $sql = $this->db->getQueryBuilder()->renameTable($table, $newName);
+        $this->requireTableSchemaRefreshment($table);
```

Comment:
I think we should drop `$_refreshTableName` and simply call `$this->db->invalidateTableSchema($table)` for the following reasons:

* The new `invalidateTableSchema()` can be used by users directly in case they use raw SQL to modify a table.
* Although methods like `renameTable()` doesn't change the table (because it doesn't execute the SQL), it is not likely that you would call this method without executing it. Moreover, even if you don't execute the SQL, the only loss is that we incorrectly invalidated the table schema and thus cause an extra table schema reloading.